app/build.gradle

```diff
@@ -136,6 +138,8 @@ dependencies {
     // Room
     implementation 'androidx.room:room-runtime:2.6.1'
     implementation 'androidx.room:room-ktx:2.6.1'
+    implementation 'com.google.firebase:firebase-auth-ktx:23.1.0'
```

Comment:
move it to Firebase section and use BOM to define automatic version management

<img width="785" alt="image" src="https://github.com/user-attachments/assets/f322833c-0209-408f-a3c9-3dc740f8c101" />